### PR TITLE
DOCS-9989 update to upgrade

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-9989 update to upgrade

### CODE REVIEW

**Comment by wwarner-salesforce** [comment]
Mysti pointed out while reviewing another PR that ACM seems to use "update" but other platform components use "upgrade", and ACM really should be made consistent.

File: modules/ROOT/pages/update-acm.adoc [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
 endif::[]
 :imagesdir: ../assets/images/
 
 Keep your Anypoint API Community Manager instance up to date to take advantage of new and improved functionality in your community.
 
-For information about updates, see xref:notifications.adoc[Important Notifications].
+For information about upgrades, see xref:notifications.adoc[Important Notifications].
 
-After you install an update with this process, it is not possible to undo the installation and return to a previous version. Follow this best-practice workflow:
+After you install an upgrade with this process, it is not possible to undo the installation and return to a previous version. Follow this best-practice workflow:
```
Comment (priyaayyar) [inline modules/ROOT/pages/update-acm.adoc]:
```suggestion
After you upgrade with this process, you cannot undo the installation and return to a previous version. Follow this best-practice workflow:
```

Comment (priyaayyar) [inline modules/ROOT/pages/update-acm.adoc]:
```suggestion
After you upgrade using this process, you cannot undo the installation and return to a previous version. Follow this best-practice workflow:
```

File: modules/ROOT/pages/update-acm.adoc [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
 +
 The staging environment will have a partial or full copy of the data in the production environment.
 . Create a short-term developer sandbox.
-. Install the update in the developer sandbox and test it.
+. Install the upgrade in the developer sandbox and test it.
 . Discard the developer sandbox.
-. Install the update in the staging sandbox and test it.
-. Install the update in your production environment.
+. Install the upgrade in the staging sandbox and test it.
+. Install the upgrade in your production environment.
```
Comment (priyaayyar) [inline modules/ROOT/pages/update-acm.adoc]:
```suggestion
. Upgrade in your production environment.
```
Install the upgrade sounds incorrect.

**Review by priyaayyar — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/update-acm.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/update-acm.adoc]: That is better! Fixed.

Comment (wwarner-salesforce) [reply modules/ROOT/pages/update-acm.adoc]: Fixed.

File: modules/ROOT/pages/update-acm.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/update-acm.adoc]: True! Changed to "install the new package."

**Review by priyaayyar — APPROVED** [review]

**Pull request merged**